estimate_length.py

```diff
@@ -205,22 +205,26 @@ def extract_pAi_from_genome(genome, window, occurences, consecutive):
             while c <= len(line)-window:
                 segment = line[c:(c+window)]
                 if consecutive*'A' in segment:
```

Comment:
ah, ok, didn't get the meaning of `consecutive` then, maybe renaming it to `n_consecutive` and/or switching the factors (`if 'A'*n_consecutive in segment:`) would help but it's clear to me now.